src/natcap/invest/spec.py

```diff
@@ -752,10 +752,11 @@ class CSVInput(FileInput):
     expected to have. The `id` of each input must match the corresponding
     column header."""
 
-    rows: typing.Union[list[Input], None] = None
```

Comment:
Now that `rows` is not an attribute, should `columns` be required instead of optional? i.e. removing `None` as the default value and acceptable type? The docstring for CSVInput could use updating as well.